app/main.py

```diff
@@ -1,25 +1,110 @@
-from abc import ABC
+from abc import ABC, abstractmethod
+from typing import Type
+
+
+class Visitor:
+
+    def __init__(
+            self,
+            name: str,
+            age: int,
+            weight: int,
+            height: int
+    ) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
+
+
+class SlideLimitationValidator(ABC):
+    @abstractmethod
+    def __init__(self, age: int, weight: int, height: int) -> None:
+        self.age = age
+        self.weight = weight
+        self.height = weight
+
+    @abstractmethod
+    def get_access(self) -> bool:
+        pass
 
 
 class IntegerRange:
-    pass
 
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+        self.status_access = None
 
-class Visitor:
-   pass
+    def __set_name__(self,
+                     owner: Type[SlideLimitationValidator],
+                     name: str) -> None:
+        self.status_access = "_" + name
 
+    def __get__(self,
+                instance: Type[SlideLimitationValidator],
+                owner: Type[SlideLimitationValidator]
+                ) -> bool:
+        return getattr(instance, self.status_access)
 
-class SlideLimitationValidator(ABC):
-    pass
+    def __set__(self,
+                instance: Type[SlideLimitationValidator],
+                value: int) -> None:
+        if not isinstance(value, int):
+            raise TypeError(
+                f"{self.status_access} should be integer,"
+                f" not a {type(value)}!"
+            )
+        setattr(
```

Comment:
validate that value in range from min_amount and max_amount, if no raise exception